### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by d9394**
1, change temporary_directory to '/dev/shm' , I feel it is faster than '/tmp'
2, add a "device" parameter to the source config and source command

### CODE REVIEW

**Comment by jketterl** [comment]
I really like the device part, but I'd like to keep the temporary directory on `/tmp`, since `/tmp/shm` does not seem to exist on all distributions. It seems to be missing both on my local machine (which is a Manjaro Linux) and on Raspbian. I pretty much started probing at that point.

It is true that using a ramdrive / tmpfs based directory is better, however I am not aware of a default location where to find one.

For the Raspberry Pi images i have included a designated "/tmp/openwebrx" tmpfs, since writing decodes to the SD card certainly decreases their live expectations.

**Comment by jketterl** [comment]
Hang on, I misread that. It's `/dev/shm` not `/tmp/shm`. I need to read up on this. Feels kind of wrong to have it on `/dev`...

**Comment by D0han** [comment]
`/dev/shm` is pretty standard to use. I would welcome such change, but please be careful to mind file sizes and their cleanup. Would be nice to put stuff in one place for cleanliness sake (like `/dev/shm/openwebrx/`).

File: owrx/source.py [hunk 1]
```diff
@@ -378,15 +378,23 @@ def activateProfile(self, profile_id=None):
 
 class RtlSdrSource(SdrSource):
     def getCommand(self):
-        return "rtl_sdr -s {samp_rate} -f {center_freq} -p {ppm} -g {rf_gain} -"
+        command = "rtl_sdr -s {samp_rate} -f {center_freq} -p {ppm} -g {rf_gain}"
+        if self.rtlProps["device"] != "0" :
```
Comment (D0han) [inline owrx/source.py]: This makes `device` key mandatory, and introduces magic value of `"0"` - can we just have it as optional key?

**Comment by d9394** [comment]
1, I think using '/dev/shm' is the easy way for change  temporary directory, if use '/dev/shm/openwebrx', it need to do 'mkdir /dev/shm/openwebrx' by your manual before running openwebrx or modify the python source code. After I running openwebrx for 2 days on my raspberry3B+, the folder '/dev/shm' only show 19 files, that is I can accept.
2, about the 'device' parameter , I want to modify again : because it was a string , so it can be fill nothing like "", that means you don't using this parameter into the command, so it looks like a option key.

**Pull request closed without merging**